HHS/Head-Start-TTADP
# remove ghost tab

## Description of change
There's no Jira for this, but it seems my plan to leave the old dashboard in place and swap it out for the new one is confusing our feature-flag users. Also, it seems we are paused on the new dashboard for now.

## How to test
Go to the regional dashboard as a user without the training_reports_dashboard feature flag. Confirm that everything looks ok.

Go to the regional dashboard as an admin user or a user with the training_reports_dashboard feature flag. Confirm that everything looks ok.

## Issue(s)

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
